.github/workflows/build.yml

```diff
@@ -15,14 +15,44 @@ jobs:
     steps:
       - name: Set env
         run: |
+          pwd
           echo "LD_LIBRARY_PATH=/usr/local/lib" >> $GITHUB_ENV
           echo "STABLE_MOSH_PATH=mosh-0.2.8-rc3" >> $GITHUB_ENV
           echo "ONIGURUMA_PATH=onig-5.9.6" >> $GITHUB_ENV
+          echo "NATURAL_DOCS_PATH=~/NaturalDocs-1.4" >> $GITHUB_ENV
+          echo "R6RS_DOC_PATH=r6rs" >> $GITHUB_ENV          
       - name: Install tools
         run: |
           sudo apt update
           sudo apt install -y gauche wget make libgmp-dev autoconf automake golang-go
-          sudo apt install -y re2c bison subversion git bear clang-tidy
+          sudo apt install -y re2c bison subversion git bear clang-tidy unzip
```

Comment:
Do we really need `subversion` here? I guess it was used to check out bootstrapped psyntax but now it's committed to the repository (and causing https://github.com/higepon/mosh/issues/40 )

Reply:
No we don't any more. Let me comment in the other thread about it.